cumulus/parachains/integration-tests/emulated/common/src/constants.rs

```diff
@@ -742,31 +400,31 @@ pub mod asset_hub_westend {
 }
 
 // Asset Hub Kusama
-pub mod asset_hub_kusama {
+pub mod asset_hub_rococo {
 	use super::*;
 	pub const PARA_ID: u32 = 1000;
-	pub const ED: Balance = parachains_common::kusama::currency::EXISTENTIAL_DEPOSIT;
+	pub const ED: Balance = parachains_common::rococo::currency::EXISTENTIAL_DEPOSIT;
 
 	pub fn genesis() -> Storage {
-		let genesis_config = asset_hub_kusama_runtime::RuntimeGenesisConfig {
-			system: asset_hub_kusama_runtime::SystemConfig {
-				code: asset_hub_kusama_runtime::WASM_BINARY
+		let genesis_config = asset_hub_westend_runtime::RuntimeGenesisConfig {
```

Comment:
Is this correct?
`mod asset_hub_rococo` vs `asset_hub_westend_runtime::RuntimeGenesisConfig/WASM_BINARY`?

Reply:
Yes. There is no asset hub rococo. It was before already using `asset-hub-kusama` for asset hub rococo.

Reply:
Oh right, it will be introduced in https://github.com/paritytech/polkadot-sdk/pull/1215
@acatangiu should include this change there.